cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code"]
+= Using Pipelines as Code
+include::_attributes/common-attributes.adoc[]
+:context: using-pipelines-as-code
+
+toc::[]
+
+:FeatureName: Pipelines as Code
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by a source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports the status.   
+
+[id="pac-key-features"]
+== Key features
+{pac} supports the following features:
+
+* Pull request status. When iterating over a pull request, the status and control run on the platform hosting the Git repository.
+* GitHub Checks API to set the status of a pipeline run, including rechecks.
+* GitHub pull request and commit events. 
+* Pull request actions in comments, such as `/retest`.
+* Git events filtering and a separate pipeline for each event.
+* Automatic task resolution in {pipelines-shortname} (local tasks, Tekton Hub, and remote URLs).
+* Use of GitHub blobs and objects API for retrieving configurations.
+* Access Control List (ACL) over a GitHub organization, or using a Prow style `OWNER` file.
+* The `tkn-pac` plugin for the `tkn` CLI tool, which you can use to manage {pac} repositories and bootstrapping.
```

Comment:
If possible, apply this user-focused wording ("which you can use to") to the other bullets in this list. It makes the list more meaningful.